<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>645</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Rockbot Editor 0.1 b6</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_map">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>390</width>
           <height>365</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Map Edit</string>
         </attribute>
         <widget class="QWidget" name="verticalWidget" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>391</width>
            <height>290</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>290</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Level:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox">
               <property name="currentIndex">
                <number>2</number>
               </property>
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetMinimumSize</enum>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QComboBox" name="mapListCombo"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>New</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollArea_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="widgetResizable">
              <bool>false</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>284</width>
                <height>75</height>
               </rect>
              </property>
              <widget class="EditorTilePallete" name="pallete" native="true">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>0</y>
                 <width>241</width>
                 <height>71</height>
                </rect>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="Terrain">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>390</width>
           <height>365</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Terrain</string>
         </attribute>
         <widget class="QListWidget" name="listWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>401</width>
            <height>361</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>Solid</string>
           </property>
           <property name="icon">
            <iconset resource="resources/icons/icons.qrc">
             <normaloff>:/toolbar_icons/Lock</normaloff>:/toolbar_icons/Lock</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Stairs</string>
           </property>
           <property name="icon">
            <iconset resource="resources/icons/icons.qrc">
             <normaloff>:/toolbar_icons/stairs.png</normaloff>:/toolbar_icons/stairs.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Door</string>
           </property>
           <property name="icon">
            <iconset resource="resources/icons/icons.qrc">
             <normaloff>:/toolbar_icons/object-order-lower.png</normaloff>:/toolbar_icons/object-order-lower.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Spikes</string>
           </property>
           <property name="icon">
            <iconset resource="resources/icons/icons.qrc">
             <normaloff>:/toolbar_icons/edit-delete.png</normaloff>:/toolbar_icons/edit-delete.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Water</string>
           </property>
           <property name="icon">
            <iconset resource="resources/icons/icons.qrc">
             <normaloff>:/toolbar_icons/flag-blue.png</normaloff>:/toolbar_icons/flag-blue.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ice</string>
           </property>
           <property name="icon">
            <iconset resource="resources/icons/icons.qrc">
             <normaloff>:/toolbar_icons/flag-green.png</normaloff>:/toolbar_icons/flag-green.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Move to Left</string>
           </property>
           <property name="icon">
            <iconset resource="resources/icons/icons.qrc">
             <normaloff>:/toolbar_icons/arrow-left.png</normaloff>:/toolbar_icons/arrow-left.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Move to Right</string>
           </property>
           <property name="icon">
            <iconset resource="resources/icons/icons.qrc">
             <normaloff>:/toolbar_icons/arrow-right.png</normaloff>:/toolbar_icons/arrow-right.png</iconset>
           </property>
          </item>
         </widget>
        </widget>
        <widget class="QWidget" name="page_npc">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="label">
          <string>NPC Edit</string>
         </attribute>
         <widget class="QListWidget" name="listWidget">
          <property name="geometry">
           <rect>
            <x>2</x>
            <y>2</y>
            <width>391</width>
            <height>191</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="editNPCButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>200</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Edit NPC</string>
          </property>
         </widget>
         <widget class="QPushButton" name="newNPCButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>200</y>
            <width>81</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>New NPC</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_Link">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Link Options</string>
         </attribute>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>371</width>
            <height>311</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QComboBox" name="link_orientation_combobox">
             <item>
              <property name="text">
               <string>Horizontal</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Vertical</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Orientation:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Size:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="link_size_spinbox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>6</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Bi-Direcional:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="link_bidi_checkbox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Is Door:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="link_isdoor_checkbox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_objects">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Object Options</string>
         </property>
         <attribute name="label">
          <string>Object Edit</string>
         </attribute>
         <widget class="QPushButton" name="editObjectButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>200</y>
            <width>101</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Edit Object</string>
          </property>
         </widget>
         <widget class="QPushButton" name="newObjectButton">
          <property name="geometry">
           <rect>
            <x>8</x>
            <y>200</y>
            <width>101</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>New Object</string>
          </property>
         </widget>
         <widget class="QListWidget" name="objectListWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>2</y>
            <width>391</width>
            <height>191</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QScrollArea" name="scrollArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>192</width>
         <height>142</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="EditorArea" name="editArea" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>211</width>
          <height>161</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionEdit_Tileset"/>
   <addaction name="actionEdit_NPC"/>
   <addaction name="actionAdd_Object"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="editToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionNormal_Edit"/>
   <addaction name="actionLock_Edit"/>
   <addaction name="actionEraser"/>
   <addaction name="actionFill"/>
   <addaction name="separator"/>
   <addaction name="actionLink"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>869</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuLevel">
     <property name="title">
      <string>Level</string>
     </property>
     <addaction name="actionOne"/>
     <addaction name="actionTwo"/>
     <addaction name="actionThree"/>
    </widget>
    <addaction name="menuLevel"/>
   </widget>
   <addaction name="menuFILE"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normalon>:/toolbar_icons/New</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/Open</normaloff>
     <normalon>:/toolbar_icons/Save</normalon>:/toolbar_icons/Open</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/Save</normaloff>:/toolbar_icons/Save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNormal_Edit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/draw-freehand.png</normaloff>
     <normalon>:/toolbar_icons/draw-freehand.png</normalon>:/toolbar_icons/draw-freehand.png</iconset>
   </property>
   <property name="text">
    <string>Normal Edit</string>
   </property>
  </action>
  <action name="actionLock_Edit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/toolbar_icons/Lock</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Lock Edit</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/toolbar_icons/draw-eraser.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/list-add.png</normaloff>:/toolbar_icons/list-add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionFill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/Fill</normaloff>:/toolbar_icons/Fill</iconset>
   </property>
   <property name="text">
    <string>Fill</string>
   </property>
  </action>
  <action name="actionLink">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/connect_no.png</normaloff>:/toolbar_icons/connect_no.png</iconset>
   </property>
   <property name="text">
    <string>Link</string>
   </property>
   <property name="toolTip">
    <string>Link Maps</string>
   </property>
  </action>
  <action name="actionStairs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/stairs.png</normaloff>:/toolbar_icons/stairs.png</iconset>
   </property>
   <property name="text">
    <string>Stairs</string>
   </property>
  </action>
  <action name="actionEdit_NPC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/im-user.png</normaloff>:/toolbar_icons/im-user.png</iconset>
   </property>
   <property name="text">
    <string>Edit NPC</string>
   </property>
  </action>
  <action name="actionEdit_Tileset">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/format-stroke-color.png</normaloff>:/toolbar_icons/format-stroke-color.png</iconset>
   </property>
   <property name="text">
    <string>Edit Tileset</string>
   </property>
  </action>
  <action name="actionLayer_Level">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/format-list-ordered.png</normaloff>:/toolbar_icons/format-list-ordered.png</iconset>
   </property>
   <property name="text">
    <string>Layer Level</string>
   </property>
   <property name="toolTip">
    <string>Layer Level</string>
   </property>
  </action>
  <action name="actionOne">
   <property name="text">
    <string>one</string>
   </property>
  </action>
  <action name="actionTwo">
   <property name="text">
    <string>two</string>
   </property>
  </action>
  <action name="actionThree">
   <property name="text">
    <string>three</string>
   </property>
  </action>
  <action name="actionAdd_Object">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/media-playlist-shuffle.png</normaloff>:/toolbar_icons/media-playlist-shuffle.png</iconset>
   </property>
   <property name="text">
    <string>Add Object</string>
   </property>
   <property name="toolTip">
    <string>Add Object</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>EditorArea</class>
   <extends>QWidget</extends>
   <header>editorarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditorTilePallete</class>
   <extends>QWidget</extends>
   <header>editortilepallete.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
